### ISSUE TITLE
Debian Docker images broken with version 3.2.2

### ISSUE THREAD

**Issue by nicolargo** [nicolargo/glances#1905]
**Describe the bug**

```
$ docker run --rm -v /var/run/docker.sock:/var/run/docker.sock:ro --pid host --network host -it nicolargo/glances:3.2.2-full
/usr/local/bin/python3: No module named glances
```

**Desktop (please complete the following information):**
 - OS: GNU/Linux Ubuntu 20.04
 - Glances Version [output of glances -V] Glances 3.2.2 (Docker images)

Introduced with https://github.com/nicolargo/glances/pull/1896.

Note: Alpine images work like a charm... So the issue only affects Debian images.

**Comment by JeremyKennedy** [comment]
Would a workaround be to use the Alpine images, or are they less featured than the Debian ones? Looking at the DockerHub tags, would `alpine-latest` be just as good as `latest`?

**Comment by auanasgheps** [comment]
> Would a workaround be to use the Alpine images, or are they less featured than the Debian ones? Looking at the DockerHub tags, would `alpine-latest` be just as good as `latest`?

That's what I used in the past, but the maintainer had issues with those as well.

My container was broken as per this issue, but switching to alpine has fixed the issue.

**Comment by imnobaka** [comment]
Switching to Alpine did not resolve the issue for me.

**Comment by golles** [comment]
> Switching to Alpine did not resolve the issue for me.

I went back to the previous version `image: nicolargo/glances:3.2.1` for me that works fine, you could try that as well

**Comment by martadinata666** [comment]
Im not using official docker image, but the custom one. Crosschecked the python3.9 libs path, so the diff is :
```
/usr/local/lib/python3.9/site-packages/
```
```
usr/local/lib/python3.9/dist-packages/
```

Can someone help test this?

**Comment by Nico83500** [comment]
Hi, same issue here and the previous version 3.2.1 works fine. Thanks for the tip.

**Comment by nicolargo** [comment]
Just to summarize...

Version 3.2.1:

* Debian image is OK
* Alpine image is NOK

Version 3.2.2:

* Debian image is NOK
* Alpine image is OK

Hope everything will be solved in version 3.2.3 :)

**Comment by JeremyKennedy** [comment]
Confirming resolved in 3.2.3!

**Comment by Nico83500** [comment]
I confirm, it works with 3.2.3, thanks ! Just to know, what is the difference between latest and 3.2.3 tags ? Hashes are not the same. Thanks !
